### PR TITLE
Views improve

### PR DESCRIPTION
**Pull request by Vlcmdc**

### CODE REVIEW

File: app/controllers/user_blog_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UserBlogController < ApplicationController
+  def index
+    @blogs = current_user.blog
```
Comment (vladimirbazhanov) [inline app/controllers/user_blog_controller.rb]: user can have 1 or many blogs?

File: app/views/layouts/_header.html.slim [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
-            => link_to "Edit profile", edit_user_registration_path
-          li
-            => link_to "Sign out", destroy_user_session_path, method: :delete
-      - else
-        ul class="nav navbar-nav navbar-right"
-          li
-            => link_to "Sign up", new_user_registration_path
-          li
-            => link_to "Sign in", new_user_session_path
+nav class="top-bar"
```
Comment (vladimirbazhanov) [inline app/views/layouts/_header.html.slim]: its the same as
```
nav.top-bar
```

File: app/views/layouts/_header.html.slim [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
+      = link_to 'Blorm', root_path
+  
+  - if user_signed_in?
+    .top-bar-left
+      ul class="menu"
+        li
+          = link_to "Profile: #{ current_user.username }", edit_user_registration_path
+    
+    .top-bar-right
+      ul class="menu"
```
Comment (vladimirbazhanov) [inline app/views/layouts/_header.html.slim]: ul.menu

File: app/controllers/user_blog_controller.rb (hunk 1 above)
Comment (Vlcmdc) [inline app/controllers/user_blog_controller.rb]: Yes, like tumblr user with many blogs

Comment (vladimirbazhanov) [inline app/controllers/user_blog_controller.rb]: So, current_user.blogs

**Pull request merged**